css/style.css

```diff
@@ -0,0 +1,2483 @@
+@font-face {
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Montserrat Regular'), local('Montserrat-Regular'),
```

Comment:
Codacy found an issue: [Expected newline after "," in a multi-line list (value-list-comma-newline-after)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)